README.md

```diff
@@ -39,12 +39,12 @@ See the [contributing guide](CONTRIBUTING.md) for information on what you can do
 * DDS library [wiki](https://en.wikipedia.org/wiki/Data_Distribution_Service)
 
 #### Geospatial Information Systems
-* OGC standards - multiple crates for standards for encoding, sharing or manipulating geospatial data [link](http://www.opengeospatial.org/standards)
-* Low level GIS functions - rust aternative to [GDAL](http://www.gdal.org/)
+* OGC standards - multiple crates for standards for encoding, sharing or manipulating geospatial data [link](http://www.opengeospatial.org/standards). There's already a create for GeoJSON but none of the others appear to have crates.
```

Comment:
Could you link the crate to GeoJSON? It's this one, right? https://crates.io/crates/geojson

Also, grammar nazism: s/craete/crate. :)